src/resolve-url.js

```diff
@@ -2,22 +2,9 @@
  * @file resolve-url.js - Handling how URLs are resolved and manipulated
  */
 
-import URLToolkit from 'url-toolkit';
```

Comment:
does URLToolkit need to be a direct dep of VHS anymore?

Reply:
it does not

Reply:
It actually isn't anymore, just a dev dependency, as it's used in tests.

Reply:
Ah, cool, missed that it was moved over.